packages/swapper/src/swappers/cow/CowSwapper.ts

```diff
@@ -76,6 +78,12 @@ export class CowSwapper implements Swapper<KnownChainIds.EthereumMainnet> {
     return cowApproveInfinite(this.deps, args)
   }
 
+  async approveAmount(): Promise<string> {
+    throw new SwapError('CowSwapper: approveAmount unimplemented', {
+      code: SwapErrorTypes.RESPONSE_ERROR
```

Comment:
Not sure that's the best error code semantically, but I don't believe adding a code for unimplemented methods is a good solution either so let's go with this!